src/test/java/bio/terra/janitor/common/configuration/TestConfiguration.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.janitor.common.configuration;
+
+import static bio.terra.janitor.common.CredentialUtils.getGoogleCredentialsOrDie;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.common.cleanup.CleanupConfig;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import java.time.Duration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/** Configs in integration test. */
+@Component
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "test")
+public class TestConfiguration {
+  /** How long to keep the resource before the 'prod' Janitor do the cleanup. */
+  public static Duration RESOURCE_TIME_TO_LIVE = Duration.ofMinutes(30);
+
+  private String resourceProjectId;
+  private String prodTrackResourceTopicId;
+  private String prodCrlClientCredentialFilePath;
```

Comment:
we probably don't want to use 'prod.' Is this the Crl client or the Janitor client?

these need javadoc comments

Reply:
I feel prod the the word most close to the use case. 
We will need to publish two copies of messages: One for test(toolsalpha), one to the tools env, I call the tools env to prod because I feel that is the 'prod' for janitor. Is there any other name I can use to differentiate them? Thanks